src/containers/Collection/CollectionList/CollectionList.tsx

```diff
@@ -91,8 +91,8 @@ export const CollectionList: React.SFC<CollectionListProps> = () => {
   );
   const [updateCollectionContacts] = useMutation(UPDATE_COLLECTION_CONTACTS, {
     onCompleted: (data) => {
-      const { numberDeleted, collectionContacts } = data.updateCollectionContacts;
-      const numberAdded = collectionContacts.length;
+      const { numberDeleted, groupContacts } = data.updateGroupContacts;
```

Comment:
We can keep const groupContacts as collectionContacts

Reply:
We are getting from the backend in this format only.
![image](https://user-images.githubusercontent.com/32592458/106565889-83099180-6555-11eb-9c68-f101418f49f3.png)

Reply:
Got it, will need to check with the backend for the migration from group to collection